tests/network-tests/src/scenarios/workingGroups.ts

```diff
@@ -8,7 +8,7 @@ import groupBudget from '../flows/working-groups/groupBudget'
 
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 scenario('Working groups', async ({ job }) => {
-  const sudoHireLead = job('sudo lead opening', leadOpening())
+  const sudoHireLead = job('sudo lead opening', leadOpening(true))
```

Comment:
This change does not help in re-hiring the leads, while full scenario is being run, i.e. the way in CI check. This code will only run if integration-tests are bring run using `./tests/network-tests/run-full-scenario.sh workingGroups`. So better to revert this change.

Reply:
@zeeshanakram3 thank you for the review. This change is not about the current failing tests but when running the `workingGroups` scenario (`./query-node/run-tests.sh workingGroups`) this job fails because `storageWorkingGroup` and `distributionWorkingGroup` leads are already hired at this point (probably from the "Init storage and distribution" job).